pages/index.md

```diff
@@ -3,21 +3,12 @@
 
 ### Status
 
-When I started this project in 2014 I was aiming to deliver a better experience when managing configuration files in JSON and other formats like YAML.
+This project is now entirely community-supported. If you would like to help or want to report an issue, please see our [GitHub](https://github.com/hjson/).
 
-But now it's time to admid that Hjson has failed to be picked up by any major projects and this is unlikely to change.
-
-Hjson is stable and you are welcome to do with it as you may wish, but because my time is limited I will no longer be updating the spec, the implementations or the syntax definitions.
-
-I would like to thank all contributors for their help! While the project was not as successful as it deserved to be, it was a great learning experience in every way.
+We are also looking for maintainers for our [.NET](https://github.com/hjson/hjson-cs), [Go](https://github.com/hjson/hjson-go), [Java](https://github.com/hjson/hjson-java), [Python](https://github.com/hjson/hjson-py) and [Ruby](https://github.com/hjson/hjson-rb) packages. If you like Hjson and want to keep it alive, please consider helping out.
 
 ```
-Goodbye Hjson!
-
-This prjoect has reached EOL.
-You are welcome to continue
-using it, just don't expect
-any updates.
+Hjson is back, baby!
```

Comment:
Back from what?  The EOL announcement was only on for a couple of days over the weekend.  This message is out of context.

Reply:
Yeah, I knew that was going to be weird, I just didn't know what to put in there.